content/documentation/Ox64/Software/Flashing.adoc

```diff
@@ -98,41 +98,41 @@ Expected result:
 
 The Bluepill is an affordable STM32 development board, based on the STM32F103C8T6 chip. We can program it to act as a USB serial adapter, just like we did with the Raspberry Pi Pico.
 
-NOTE: The one catch is that you already need a serial adapter in order to program your Bluepill board. The good news is that you serial adapter does **not** have to be one from from the link:/documentation/Ox64/Further_information/Compatible_UARTs/[Compatible_UARTs] list. If you own an SWD-capable debugger (ST-Link, J-link, etc.), you can use that for programming the Bluepill as well.
+NOTE: The one catch is that you already need a serial adapter in order to program your Bluepill board. The good news is that you serial adapter does **not** have to be one from from the link:/documentation/Ox64/Further_information/Compatible_UARTs/[Compatible_UARTs] list. If you own an SWD-capable debugger (FT232RL, ST-Link, J-link, etc.), you can use that for programming the Bluepill as well.
```

Comment:
FT232RL isn't a SWD debugger.

Reply:
https://github.com/pine64/website/pull/59/commits/c3021931de513b7a1784ef41957d62fdd3fa0880

Reply:
The second sentence talks about serial adapters; the third sentence talks about SWD debuggers. Hence the reference to the FT232RL belongs in the second sentence ..